src/compiler/diagnosticMessages.json

```diff
@@ -836,6 +836,10 @@
         "category": "Error",
         "code": 2307
     },
+    "An `export * from {0}` declaration has already exported a member named '{1}'. Consider explicitly re-exporting to resolve the ambiguity.": {

```

Comment:
Module {0} has already exported a member named '{1}'. Consider explicitly re-exporting to resolve the ambiguity."